src/lib/Scenes/Artwork/Components/PartnerCard.tsx

```diff
@@ -86,7 +86,7 @@ export class PartnerCard extends React.Component<Props, State> {
       return null
     }
     const { isFollowedChanging } = this.state
-    const imageUrl = partner.profile ? partner.profile.icon.url : null
+    const imageUrl = partner.profile && partner.profile.icon ? partner.profile.icon.url : null
```

Comment:
You can also use the `get` util here, I believe!

Reply:
Personally would rather not add overhead but personally preference.

Reply:
`get()` also doesn't provide any kind of type safety 😞 We'll get `strictNullChecks` turned on here eventually 😅

Reply:
Ah, I didn't mean importing directly from lodash.

We built a [typesafe version](https://github.com/artsy/emission/blob/master/src/lib/utils/get.ts) of this function in reaction, and it was imported here in April. 😄

Reply:
Oh, super-cool!